api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -755,4 +761,172 @@ public void copyFile_shouldCopyInputstreamToOutputstreamAndCloseTheOutputstream(
 		verify(output, times(1)).close();
 	}
 
+	/**
+	 * @see OpenmrsUtil#isTestMode()
+	 */
```

Comment:
please remove all the javadocs above the tests you added. it just clutters the test class and does not add value. we can see what you tested in the test itself and navigate to the method easily.